src/dataframe/dataframe.jl

```diff
@@ -1138,11 +1138,14 @@ function categorical!(df::DataFrame,
 end
 
 """
-    append!(df1::DataFrame, df2::AbstractDataFrame)
+    append!(df1::DataFrame, df2::AbstractDataFrame, cols::Symbol=:setequal)
 
 Add the rows of `df2` to the end of `df1`.
 
-Column names must be equal (including order), with the following exceptions:
+Column names must be equal, including order, if `cols` argument is `:orderequal`.
```

Comment:
Better document the default behavior (`:setequal`) first? Maybe take the same list as for `push!`.

Reply:
right. fixed.

Reply:
Ah - the list is different, because we currently do not provide all the options that `push!` provides. This is left for the future PR.